### ISSUE TITLE
Change in note for Export reports to PowerPoint

### ISSUE THREAD

**Issue by prvillal** [MicrosoftDocs/powerbi-docs#3286]
[Enter feedback here]
Please change the Note: PowerPoint adds alt text for each item on the page. The report creator can include alt text when designing the report. Otherwise, the default is "No alt text provided."

To be read as follow: 
PowerPoint includes the name of each visual on the page and adds alt text for each item. The report creator can include alt text when designing the report. Otherwise, the default is "No alt text provided."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0df1c4c0-80f9-f71a-175f-8362a29ad597
* Version Independent ID: c2a64e2a-ce65-dfb7-588a-381c145ab193
* Content: [Export entire reports to PowerPoint - Power BI](https://docs.microsoft.com/en-us/power-bi/consumer/end-user-powerpoint)
* Content Source: [powerbi-docs/consumer/end-user-powerpoint.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/consumer/end-user-powerpoint.md)
* Service: **powerbi**
* Sub-service: **pbi-explore**
* GitHub Login: @mihart
* Microsoft Alias: **mihart**

**Comment by mihart** [comment]
Hi @prvillal , 
Thank you for your suggestion.  We've made the update and set the article to republish. 
All the best,
The DOCS team.
